packages/vhd-lib/Vhd/VhdDirectory.js

```diff
@@ -281,7 +281,7 @@ exports.VhdDirectory = class VhdDirectory extends VhdAbstract {
       }
     }
     setBitmap(this.#blockTable, blockId)
-    return sectorsToBytes(this.sectorsPerBlock)
+    return this._handler.getSizeOnDisk(this._getFullBlockPath(blockId))
```

Comment:
I think the mergeBlock should return the size change on the parent vhd 
If the block exists, it should be "previous size" minus "new size", and the vhdabstract.mergeBlock method should be updated accordinlgy

Reply:
As we discussed it previously, I also agree it would be good.
However I think it belongs to another card, and I think we'll make another PR for this to avoid straying too much from the task.